src/app/body/cart/product-card/product-card.component.scss

```diff
@@ -0,0 +1,122 @@
+.product-card {
+  display: flex;
+  border-bottom: 1px solid #F9F9F9;
+  padding-bottom: 22px;
+  margin-top: 32px;
+  &_right {
+    flex-grow: 1;
+    max-width: 100px;
+    margin-right: 16px;
+    button {
+      border: none;
+      background: none;
+      font-size: 12px;
+      color: #A9A9A9;
+      cursor: pointer;
+      transition: .3s;
+      img {
+        margin-right: 10px;
+      }
+      &:hover {
+        background: #F9F9F9;
+      }
+    }
+  }
+  &_image {
+    width: 100%;
+    max-height: 67px;
+    object-fit: cover;
+    border-radius: 12px;
+  }
+  &_content {
+    flex-grow: 1;
+  }
+  &_title {
+    font-weight: 500;
+    font-size: 15px;
+    &:first-letter {
+      text-transform: uppercase;
+    }
+  }
+  &_info {
+    font-size: 12px;
+    &-item {
+      margin-top: 3px;
+      span:first-child {
+        display: inline-block;
+        color: #A9A9A9;
+        width: 80px;
+      }
+    }
+  }
+  &_rating {
+    margin: 15px 0;
+  }
+  &_price {
+    display: flex;
+    justify-content: space-between;
+    position: relative;
+    .price {
+      font-weight: 600;
+      font-size: 18px;
+      transition: .3s;
+      &_calc {
+        position: absolute;
+        font-size: 10px;
+        color: #A9A9A9;
+        top: -10px;
+      }
+    }
+    .count {
+      display: flex;
+      align-items: center;
+      background: #F9F9F9;
+      border: 1px solid #D1D1D1;
+      border-radius: 12px;
+      height: 32px;
+      padding: 0 5px;
+      input, select {
+        border: none;
+        background: none;
+        font-weight: 600;
+        font-size: 12px;
+      }
+      input {
+        padding: 0 5px;
+        max-width: 40px;
+        text-align: center;
+        &::-webkit-inner-spin-button, &::-webkit-outer-spin-button {
```

Comment:
What does it means?

Reply:
to hide default browser arrows of input[type=number]